projects/storefrontlib/src/lib/ui/pages/pages.module.ts

```diff
@@ -1,69 +0,0 @@
-import { CommonModule } from '@angular/common';
-import { NgModule } from '@angular/core';
-import { RouterModule } from '@angular/router';
-import { AuthGuard, ConfigModule } from '@spartacus/core';
-import { LogoutGuard } from '../../../cms-components/index';
-import { CmsPageGuard } from '../../../cms-structure/guards/cms-page.guard';
-import {
-  PageLayoutComponent,
-  PageLayoutModule,
-} from '../../../cms-structure/page/index';
-import { CartPageModule } from './cart-page/cart-page.module';
-import { OrderConfirmationPageModule } from './order-confirmation-page/order-confirmation-page.module';
-import { ProductPageModule } from './product-page/product-page.module';
-import { defaultRoutingConfig } from './default-routing-config';
-
-const pageModules = [
-  CartPageModule,
-  OrderConfirmationPageModule,
-  ProductPageModule,
-];
-
-@NgModule({
-  imports: [
-    ConfigModule.withConfig(defaultRoutingConfig),
-    CommonModule,
-    ...pageModules,
-    PageLayoutModule,
-    RouterModule.forChild([
-      {
-        // This route can be dropped only when we have a mapping path to page label for content pages
-        path: null,
-        canActivate: [CmsPageGuard],
-        component: PageLayoutComponent,
-        data: { pageLabel: 'homepage', cxRoute: 'home' },
-      },
-      {
-        path: null,
-        canActivate: [LogoutGuard],
-        component: PageLayoutComponent,
-        data: { cxRoute: 'logout' },
-      },
-      {
-        path: null,
-        canActivate: [CmsPageGuard],
-        component: PageLayoutComponent,
-        data: { pageLabel: 'search', cxRoute: 'search' },
-      },
-      {
-        path: null,
-        canActivate: [CmsPageGuard],
-        component: PageLayoutComponent,
-        data: { cxRoute: 'category' },
```

Comment:
We need to bring this route declaration back to have a configurable route for category page (i.e. `my/category/123`).